pkg/process/procutil/process_linux.go

```diff
@@ -175,6 +175,38 @@ func (p *Probe) ProcessesByPID(now time.Time) (map[int32]*Process, error) {
 	return procsByPID, nil
 }
 
+// StatsWithPermByPID returns the stats that require extra permission to collect for each process
+// the parameter `returnZeroVals` controls whether to include all zero stats or not in the return map,
+// not returning all zero stats would help process-agent to deserialize less data
```

Comment:
Will we ever want `returnZeroVals` to be true?

Reply:
Not currently, but was thinking `procutil` acts as a library should have the option to return zero values. Is that OK?

Reply:
I don't think I have enough contextual knowledge to know when a caller might want it, but yeah it seems fine